lib/MusicBrainz/Server/Edit/CheckForConflicts.pm

```diff
@@ -123,7 +123,7 @@ sub merge_changes {
                 );
 
             # Stores a mapping of the JSON serialization to the object value
-            my %hashed = map { @$_ } ($new, $ancestor, $current);
```

Comment:
And to be clear, I had changed the order to be the same as `merge` below, but there was no actual reason to.